src/BlockTypes/MiniCart.php

```diff
@@ -219,6 +226,34 @@ protected function get_markup() {
 			$cart_contents_total += $cart->get_subtotal_tax();
 		}
 
+		$wrapper_classes = 'wc-block-mini-cart wp-block-woocommerce-mini-cart';
+		$classes         = '';
+		$style           = '';
+
+		if ( ! isset( $attributes['transparentButton'] ) || $attributes['transparentButton'] ) {
+			$wrapper_classes .= ' is-transparent';
+		}
+
+		if ( ! empty( $attributes['textColor'] ) ) {
+			$classes .= sprintf(
+				' has-%s-color has-text-color',
+				esc_attr( $attributes['textColor'] )
+			);
+		} elseif ( ! empty( $attributes['style']['color']['text'] ) ) {
+			$style   .= 'color: ' . esc_attr( $attributes['style']['color']['text'] ) . ';';
+			$classes .= ' has-text-color';
+		}
```

Comment:
Will it be possible to use some of the methods introduced in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5133 `StyleAttributesUtils` here?

No need to do it now, but might be a good follow-up issue.

cc @gigitux

Reply:
Sure, I added a todo to address your comment once the 5133 is merged.